File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cs
using System;

public class Solution {
    public int solution(int num, int k) {
        string s = num.ToString();
        for(int i = 0; i < s.Length; i++)
        {
            if(int.Parse(s[i].ToString()) == k) return i + 1;
        }
        return -1;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int solution(int num, int k) {
    string s = to_string(num);
    for(int i = 0; i < s.size(); i++)
    {
        if(s[i] == k + '0')
        {
            return i + 1;
        }
    }
    return -1;
}